.travis.yml

```diff
@@ -1,12 +1,35 @@
+---
 language: node_js
 node_js:
```

Comment:
I used the default as generated by ember-cli (plus I added the extra versions of ember) which I'm not sure that it should mirror the one from ember-cli itself. Maybe it should mirror the one from the generator at master? IDK.